src/app/users/users.component.html

```diff
@@ -1 +1,20 @@
-<p>users works!</p>
+<br />
+<div *ngFor="let u of users" class="card" style="width: 18rem">
```

Comment:
let u of users 

never use a letter for the name of a variable.

just do something like
let user of users

Reply:
ok fixed thanks!

Reply:
fixed thanks